a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -551,8 +551,7 @@ <h3>Introduction to key accessibility information</h3>
 			<div class="note">
 				<p>This document does not define the order in which to
 					show the key accessibility information; each
-					implementer can decide the preferred order for
-					showing the accessibility information that
+					implementer can decide the preferred order and appropriate headings to use to show the accessibility information that
```

Comment:
Not sure then if it makes sense to include headings in the localisation file as it would introduce confusion (i.e. use only part of the localisation and provide a second localisation to refine implementation with our own heading structure).

Reply:
@gautierchomel I brought up this same question if it should just be "images" instead of "complex images" however regular simple images do not need anything other than an alt text description. This is why we chose "complex images" is there a better word to describe images needing more than just a simple alt text description?

Reply:
I disagree, a content creator can choose to add an extended description on a "simple" image, because of the context or will to see it displayed. Nothing prevents him to do so.